net-libs/cppzmq/cppzmq-4.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="High-level CPP Binding for ZeroMQ"
+HOMEPAGE="https://github.com/zeromq/cppzmq"
+SRC_URI="https://github.com/zeromq/${PN}/archive/v${PV}.tar.gz ->  ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+# Except for test these are used to alter installed cmake files:
+IUSE="test drafts"
+REQUIRED_USE="drafts? ( test )"
+RESTRICT="!test? ( test )"
+
+RDEPEND=">=net-libs/zeromq-4.3.1"
+# Tests require cmake modules from catch2 and headers from older version of catch
```

Comment:
That just makes no sense. Also, didn't you introduce this inconsistency yourself?

Reply:
That's how they do it unfortunately. catch2 provides cmake modules to simplify testing but tests themselves are not compatible with catch2 headers. Upstream's workaround is to download catch2 module during cmake run...

Reply:
The solution is strange but it works. Alternative solution would be to edit <catch-2` ebuild and force it to install cmake module. It could be the case, that they ported it back to 1.x branch, but it is not in the ebuild